### ISSUE TITLE
Option to specify pages to render

### ISSUE THREAD

**Issue by cweiske** [bash0/cewe2pdf#83]
For debugging purposes it would be nice if cewe2pdf had a command line option to specify the pages to render.

Many cewe photo book products have a minimum number of pages, but I want to test/debug rendering on page 2 only. cewe2pdf currently renders all pages, even if they are empty. This takes time and slows down development.

**Comment by AnEnglishmanInNorway** [comment]
I agree, this would be useful. I've similarly wasted time trying to debug a
font issue a long way out in one of my albums (an issue which I failed to
reproduce in a smaller album because the XML for text varies considerably
depending on how the user has selected and changed the characteristics of
sections of text).
I have one small worry, however, based on my vague memory of the work I did
to improve text handling. I have a nasty feeling that the code (the XML
itself) makes assumptions about what has come before (for example, what is
the current font, current size, or similar). This is a very wishy washy
concern, and is probably not be enough to stop us implementing a simple
page selector argument, but I don't think I would want to guarantee, as
part of the description of a page selector option, that processing just
page 99 of an album would produce an identical output page to the page 99
which was preceded by all the other pages!
